### ISSUE TITLE
Blocking CRAN release: S3 generic/method consistency ... NOTE Mismatches for apparent methods not registered:

### ISSUE THREAD

**Issue by talgalili** [talgalili/gplots#21]
from win build

```
* checking S3 generic/method consistency ... NOTE
Mismatches for apparent methods not registered:
getVennCounts:
  function(l, universe, verbose, ...)
getVennCounts.list:
  function(l, universe, verbose, intersections)
See section 'Registering S3 methods' in the 'Writing R Extensions'
manual.
```

**Comment by talgalili** [comment]
@arni-magnusson - before I can ship gplots to CRAN, this needs to be fixed.
Any chance you could take a look at this and submit a PR?

(in general, moving to roxygen2 would greatly improve our quality of life with long term maintenance of the package. If you'll get to it, I'll gladly review that PR)

**Comment by arni-magnusson** [comment]
I've taken a close look at this issue, done some experiments, including submissions to win-builder.r-project.org. As far as I can tell, the issue was fixed with your commit e03333e. 

In the note quoted above, the ellipsis was missing in the getVennCounts.list() method, but the master branch now has the required ellipsis.

As for roxygen2, the same thought has entered my mind as the maintainer of [gdata](https://github.com/r-gregmisc/gdata), but no... I'd also be happy to review that PR :)

**Comment by talgalili** [comment]
Got it, thanks for checking :)
